PrebidMobile/PrebidServerAdapter/src/main/java/org/prebid/mobile/prebidserver/PrebidServerAdapter.java

```diff
@@ -440,6 +476,16 @@ private JSONObject getUserObject() {
             }
         } catch (JSONException e) {
         }
+        try {
```

Comment:
Let's decide on whether to do this only for FB demand enabled or every time if the class exists.